packages/admin-panel/.eslintrc

```diff
@@ -3,6 +3,10 @@
     "@beyondessential/eslint-config-beyondessential",
     "plugin:prettier/recommended"
   ],
+  "rules": {
+    "react/jsx-props-no-spreading": "off",
+    "react/jsx-one-expression-per-line": "off"
```

Comment:
I agree with those rule overrides! 👍  I will add them as a suggestion to our updated style guide